test/test_jit.py

```diff
@@ -19572,8 +19572,37 @@ def wrong4(x):
                 # type: (OneTwoWrong) -> int
                 return as_interface(x)
 
-    # TODO test: interface-interface class-interface inheritance errors,
-    # NamedTuple inheritance errors
+        # Test interface/class python assignment
+        with torch.jit._disable_emit_hooks():
```

Comment:
_disable_emit_hooks?

Reply:
Is there a bug in current serialization? I would imagine this would already work.

Reply:
This should be solved by https://github.com/pytorch/pytorch/pull/26770, it's not really a bug but a missing feature, we could be able to serialize the interface type but the run-time object that backed the interface does not get properly serialized, so it missed the `Foo` class when serializing.